### ISSUE TITLE
Rename to karma

### ISSUE THREAD

**Issue by jperl** [dignifiedquire/grunt-testacular#9]
Testacular has been renamed to [Karma](karma-runner.github.com)

**Comment by zilvinasu** [comment]
@jperl Not sure if I am lucky or the tests could still be run via grunt-testacular? Find it weird that I removed testacular, and the grunt task still worked.

**Comment by dignifiedquire** [comment]
grunt-testacular has the old testacular as a dependency so it still has a version of testacular even if you removed it from your top level node_modules. 

I won't rename and migrate this package as we have decided that one grunt task is enough. But I will work with @geddesign to bring the missing functionality into the official grunt task.

**Comment by jperl** [comment]
Awesome, I definitely agree that one project is better. So this project will be deprecated when https://github.com/karma-runner/grunt-karma catches up?

**Comment by artyomtrityak** [comment]
Guys please fix blocking grunt run for grunt-karma. Running in the another terminal is pain.

**Comment by geddski** [comment]
you don't need to run in another terminal for single-run tests: https://github.com/karma-runner/grunt-karma#single-run

Also please post issues on the [grunt-karma](https://github.com/karma-runner/grunt-karma) tracker.

**Comment by artyomtrityak** [comment]
But this case on each file change all browsers will be started... each time. It's not good

**Comment by geddski** [comment]
@artyomtrityak agreed, when using with watch it's better not to do single-run. @Dignifiedquire and I can look into adding this to [grunt-karma](https://github.com/karma-runner/grunt-karma).

**Comment by artyomtrityak** [comment]
The main problem now that if i run Karma not in singleRun it blocks grunt watch.
But if i run in singleRun it starts and stops browsers after each file change.

In 0.6.x versions Testacular allows run watch mode for grunt and watch for Testacular so browsers are not stopping.

Thanks!

**Comment by dignifiedquire** [comment]
@artyomtrityak This has nothing to do with the version of Testacular/Karma. It's the way I implemented this task vs how grunt-karma aka gruntacular is implemented. If you want you can just fork this task and update the dependency to karma@0.8 and it will work as expected.

**Comment by geddski** [comment]
Hence the benefits of collaborating on a single grunt task.

**Comment by vojtajina** [comment]
I think it's great that we decided to merge both plugins into single `grunt-karma`, @Dignifiedquire can you put some notice to the readme, so that people don't get confused and rather use `grunt-karma`?
